web/react/components/post.jsx

```diff
@@ -223,6 +244,10 @@ export default class Post extends React.Component {
                                 posts={posts}
                                 handleCommentClick={this.handleCommentClick}
                                 retryPost={this.retryPost}
+                                setHasEmbedState={this.setHasEmbedState}
+                                toggleEmbedVisibility={this.toggleEmbedVisibility}
+                                hasEmbed={this.state.hasEmbed}
+                                embedVisible={this.state.embedVisible}
```

Comment:
I think it's better to not overdesign here and keep it all in PostBody if that's all the current UI requires. It'll keep it simple and if it needs to be accessed from another component later on then we can deal with that case